packages/jest-editor-support/package.json

```diff
@@ -11,7 +11,8 @@
     "typescript": "^2.0.10"
   },
   "dependencies": {
-    "babylon": "^6.13.0"
+    "babylon": "^6.13.0",
+    "astroll": "^1.0.0"
```

Comment:
a 10 line ast walker that keeps track of the parents:
https://github.com/kentaromiura/astroll/blob/master/index.js#L15-L32